robottelo/vm.py

```diff
@@ -448,6 +486,40 @@ def remove_katello_ca(self):
         except AssertionError:
             raise VirtualMachineError('Failed to remove the katello-ca rpm')
 
+    def remove_capsule_katello_ca(self, capsule=None):
+        """Removes katello-ca rpm from the virtual machine.
+
+        :param: str capsule: Capsule hostname
+        :return: None.
+        :raises robottelo.vm.VirtualMachineError: If katello-ca wasn't removed.
+        """
+        # Not checking the return_code here, as rpm can be not even installed
+        # and deleting may fail
+        ssh.command(
+            'yum erase -y $(rpm -qa |grep katello-ca-consumer)',
+            self.ip_addr
+        )
+        # Checking the return_code here to verify katello-ca rpm is actually
+        # not present in the system
+        result = ssh.command(
+            'rpm -q katello-ca-consumer-{0}'.format(capsule), self.ip_addr)
+        if result.return_code == 0:
+            raise VirtualMachineError('Failed to remove the katello-ca rpm')
+        # Resetting rhsm.conf to point to cdn
```

Comment:
I think all this below part has to be the part of a separate function.

Please create a separate function just to reset rhsm conf.

Reply:
@jyejare I kept that as part of this function because it's not using that much frequent(or by other function) .
rest all changes are in place , please take a look